terraform/modules/iam_user/ecr_user/policy.json

```diff
@@ -0,0 +1,12 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": [
+        "ecr:*"
```

Comment:
Do we want to limit these permissions?

Reply:
I'll limit it when I learn more about permissions and such :)

Reply:
👍